common-util/src/main/java/org/sunbird/common/request/UserRequestValidator.java

```diff
@@ -366,6 +369,29 @@ public void validateUpdateUserRequest(Request userRequest) {
     validateFrameworkDetails(userRequest);
   }
 
+  private void validateUserOrgField(Map<String, Object> request) {
+    if (request.get(JsonKey.ORGANISATIONS) instanceof List) {
+      List<Object> list = (List<Object>) request.get(JsonKey.ORGANISATIONS);
+      for (Object map : list) {
+        if (!(map instanceof Map)) {
+          throwInvalidUserOrgData();
+        }
+      }
+
+    } else if (request.get(JsonKey.ORGANISATIONS) != null) {
+      throwInvalidUserOrgData();
+    }
+  }
+
+  private void throwInvalidUserOrgData() {
+    ProjectCommonException.throwClientErrorException(
+        ResponseCode.dataTypeError,
+        MessageFormat.format(
```

Comment:
can you print the final message -  that following format function outputs

Reply:
Data type of organisations should be List.map

Reply:
Data type of organisations should be List.map

Reply:
Message -> Organisations should be of type - list of map